doc/api/stream.md

```diff
@@ -1231,65 +1228,66 @@ would be piped into the parser, which is a more idiomatic Node.js stream
 approach.
 
 ```javascript
-const util = require('util');
 const Transform = require('stream').Transform;
-util.inherits(SimpleProtocol, Transform);
 
-function SimpleProtocol(options) {
-  if (!(this instanceof SimpleProtocol))
-    return new SimpleProtocol(options);
+class SimpleProtocol extends Transform {
+  constructor(options) {
+    super(options);
+    if (!(this instanceof SimpleProtocol))
+      return new SimpleProtocol(options);
```

Comment:
Here too.  I'll stop pointing it out from now on but if there are more instances, please fix those as well.